spec/truffle/arguments_descriptor_spec.rb

```diff
@@ -15,16 +15,24 @@ def self.arguments
 end
 
 module ArgumentsDescriptorSpecs
-  Info = Struct.new(:values, :descriptor, :arguments)
+  class Info
+    attr_reader :values, :descriptor, :arguments
+
+    def initialize(values, descriptor, arguments)
+      @values = values
+      @descriptor = descriptor
+      @arguments = arguments
+    end
+  end
 end
 
 describe "Arguments descriptors" do
   def truffleruby?
     defined?(::TruffleRuby)
   end
 
-  def info(*args)
```

Comment:
There was no keyword arguments involved in `*args` or `Struct.new(:values, :descriptor, :arguments)` AFAIK.
I guess you mean in the implementation of `Struct.new` itself (`def self.new(klass_name, *attrs, keyword_init: false, &block)`).
Seems a OK change if it's helpful.

Reply:
Yes it's `keyword_init` etc. This is just much simpler for me to debug.